src/index.d.ts

```diff
@@ -18,7 +19,7 @@ import {
   ActivityIndicatorProperties,
   SwitchProperties,
   StatusBarStyle,
-  ButtonProps as NativeButtonProps,
+  ModalProps,
```

Comment:
Why'd you add this line here?

Reply:
kinda weird because [here](https://github.com/react-native-training/react-native-elements/blob/next/src/index.d.ts) ModalProps import is already there

Reply:
on master I see "ButtonProps as NativeButtonProps" indeed, but on next branch it's "ModalProps"